src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [film, findFilm] = useState('');
+
+  const moviesFilter = moviesFromServer.filter((movie) => {
+    const { description, title } = movie;
```

Comment:
You also can use destruction in params
```suggestion
  const moviesFilter = moviesFromServer.filter(({ title, description }) => {
```